playground/infrastructure/helm-playground/templates/deployment-router.yml

```diff
@@ -40,6 +40,27 @@ spec:
            value: "remote"
          - name: CACHE_ADDRESS
            value: {{ .Values.redis_ip }}
+         - name: NUM_PARALLEL_JOBS
+           value: "5"
+       livenessProbe:
+         httpGet:
+           path: /liveness
+           port: 8080
+         initialDelaySeconds: 3
+         periodSeconds: 3
+       readinessProbe:
+         httpGet:
+           path: /readiness
+           port: 8080
```

Comment:
Similar to the comment above, can we populate the port parameter from a helm variable?

Reply:
Fixed. Everything was moved to the values.yaml file